kibbeh/src/modules/layouts/ElectronHeader.tsx

```diff
@@ -49,7 +49,7 @@ function WinHeader() {
           }}
         />
         <WinButton
-          icon={<WinMinimizeIcon width={10} height={10} />}
+          icon={<WinMinimizeIcon width={10} height={1} />}
```

Comment:
don't keep the height `1` remove the height customization so it uses the svg's original height

Reply:
sure!